### ISSUE TITLE
aria-labelledby prop doesn't work for ComboBox

### ISSUE THREAD

**Issue by matthewsaunderson** [microsoft/fluentui#21520]
### Environment Information

- **Package version(s)**: FluentUI React 8.52.1

### Please provide a reproduction of the bug in a codepen:
https://codepen.io/matthewsa/pen/QWONyPK

#### Actual behavior:
The aria-labelledby prop gets passed to the root element, instead of the combobox input element. This causes two issues with Accessibility Insights: the combobox doesn't have a label, and the root container element has an aria attribute that it isn't allowed.

#### Expected behavior:
The aria-labelledby prop gets passed to the input element, instead of its container. This is what happens with all the other components on my form.

### Priorities and help requested:

Low Priority - I've worked around the problem by using the ariaLabel prop and duplicating the information in the label component I was trying to reference.

**Comment by micahgodbolt** [comment]
same issue for aria-describedby
